cmd/ocm/create/cluster/cmd.go

```diff
@@ -292,6 +347,35 @@ func run(cmd *cobra.Command, argv []string) error {
 	return nil
 }
 
+func osdProviders() []string {
+	return []string{"aws", "gcp"}
+}
+
+func minComputeNodes() (min int) {
```

Comment:
let's avoid having this calculation in the client side. we can provide the required info but I don't want to maintain this code here

Reply:
What should I do instead?

Do you mean we should add an API returning minimum rules?  Or returning minimum given other fields?  That actually sounds like a good use for starting to implement structured errors!  We would send a (dryRun to be safe) request with nodes = 0 and get something like:
```json
{
  "kind": "Error",
  ...
  "fields": {
    "compute.nodes": {
      "message": "Multi AZ clusters require at least 9 compute nodes.",
      "min": 9
    },
    ... possibly errors for other missing/bad fields ...
  }
}
```

But I don't want to block this PR until we add an API :motorway: 
So, what should I do meanwhile?

Client-side validation of minimum is not a must — the server will error if too low.  However, if that happens, we don't get a structured error pointing to specific field, so hard to re-prompt.  All I can do is show the error and exit, and user has to answer all prompts again :tired_face: (one downside to prompts compared to command-line flags).
I also used the minimum as a default.  I don't think a "high enough for all cases" default like 9 would make sense, so I'd just leave the field blank instead, with no hint what's the valid range?  User can type `?` for help, but are you OK with maintaining the minimums rules in the flag help text?

IMHO, if we take the hard position and remove minimums info from the help text and remove defaults and validation, the experience becomes too bad.  It's the same logic that made us do it in the web UI...

Reply:
Structured errors are tracked internally in https://issues.redhat.com/browse/SDA-2713, added this use case there.
Presently, I don't see a way to give acceptable UX without duplicating this information here.

Updated the flag's help text to also use this function so at least it's centralized.